docs/openstack-cloud-controller-manager/expose-applications-using-loadbalancer-type-service.md

```diff
@@ -236,6 +236,13 @@ Request Body:
   This annotation is automatically added and it contains the floating ip address of the load balancer service.
   When using `loadbalancer.openstack.org/hostname` annotation it is the only place to see the real address of the load balancer.
 
+- `loadbalancer.openstack.org/custom-tags`
+
+  Allows customable loadbalancer tag configurable added during initial stage of loadbalancer creation.
+  Tags are abitrary strings that can me added to loadbalancer and using this annotation allows adding of one or more custom tag to the `LoadBalancer`, `Listener` and `Pool`
```

Comment:
I noticed that you add all of the tags to all of kinds resources of the LB associate with the Service. Why not split it to different annotation?  Just like: `loadbalancer.openstack.org/loadbalancer-tags`, `loadbalancer.openstack.org/listener-tags` and `loadbalancer.openstack.org/pool-tags`.

Reply:
I specifically asked to not do that, I do not think we should end up with 10 more annotations supported for each of the resource we create in CPO.

Reply:
> I specifically asked to not do that, I do not think we should end up with 10 more annotations supported for each of the resource we create in CPO.

Why shouldn't we do that.

According to the [issue descriptoin](https://github.com/kubernetes/cloud-provider-openstack/issues/2327#issue-1856603420), these tags maybe used to config some special functions. So, I really don't think we can config them arbitrary. We couldn't ensure what will happen if we add a tag that originally used to config listener's function to pool. Because of we don't know Octavia driver how to use it,  especially when cloud provider has a custom Octavia driver.

Another possible solution could be to add a special prefix to classify tags. Such as: 
```
loadbalancer.openstack.org/custom-tags: "listener-<tag1>,pool-<tag2>"
```

How do you think?

Reply:
FYI: https://github.com/kubernetes/cloud-provider-openstack/issues/2327#issuecomment-1947301740

I understand it would mean complete rework, but IMHO:
1. it would solve this issue in more elegant fashion
2. there is more room for accomodating future use-cases (e.g. adding more options / adding new tag types)